.ci/vfio_jenkins_job_build.sh

```diff
@@ -70,6 +72,13 @@ create_user_data() {
 		dnf_proxy="dnf:
   https_proxy: ${https_proxy}
   proxy: ${http_proxy}"
+		docker_proxy='[Service]
+    Environment="HTTP_PROXY='${http_proxy}'" "HTTPS_PROXY='${https_proxy}'" "NO_PROXY='${no_proxy}'"'
```

Comment:
indentation issue?

Reply:
nop, the yaml where this variable is used requires these spaces